src/main/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/LdapRegistrationDAO.scala

```diff
@@ -213,6 +213,6 @@ class LdapRegistrationDAO(
   }
 
   override def setUserAzureB2CId(userId: WorkbenchUserId, b2CId: AzureB2CId, samRequestContext: SamRequestContext): IO[Unit] =
-    executeLdap(IO(ldapConnectionPool.modify(userDn(userId), new Modification(ModificationType.ADD, Attr.azureB2CId, b2CId.value))), "setUserAzureB2CId", samRequestContext)
+    executeLdap(IO(ldapConnectionPool.modify(userDn(userId), new Modification(ModificationType.REPLACE, Attr.azureB2CId, b2CId.value))), "setUserAzureB2CId", samRequestContext)
```

Comment:
I figure `REPLACE` semantics is safe for LDAP because:
a) this always happens after the postgres `setUserAzureB2CId`, which should fail if we're trying to change a pre-existing value.
b) the ldap stuff is temporary anyway